### PR TITLE
PAYARA-1912 MicroProfile Fault Tolerance 1.0

### PR DESCRIPTION
**Pull request by Pandrex247**
Initial stab at implementing MicroProfile Fault Tolerance 1.0

### CODE REVIEW

**Comment by Pandrex247** [comment]
Ready for some savaging :)

**Comment by Pandrex247** [comment]
@smillidge Basic asadmin commands added.

**Comment by Pandrex247** [comment]
Jenkins test please

File: appserver/payara-appserver-modules/microprofile/fault-tolerance/src/main/java/fish/payara/microprofile/faulttolerance/admin/SetFaultToleranceConfigurationCommand.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/*
```
Comment (michaelranaldo) [inline appserver/payara-appserver-modules/microprofile/fault-tolerance/src/main/java/fish/payara/microprofile/faulttolerance/admin/SetFaultToleranceConfigurationCommand.java]: License header is missing

File: appserver/payara-appserver-modules/microprofile/fault-tolerance/src/main/java/fish/payara/microprofile/faulttolerance/admin/GetFaultToleranceConfigurationCommand.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
+/*
```
Comment (michaelranaldo) [inline appserver/payara-appserver-modules/microprofile/fault-tolerance/src/main/java/fish/payara/microprofile/faulttolerance/admin/GetFaultToleranceConfigurationCommand.java]: License header is missing

**Comment by Pandrex247** [comment]
Jenkins test please

**Pull request merged**